packages/tools/bundleDefaultPlugins.test.ts

```diff
@@ -0,0 +1,93 @@
+import path = require('path');
+import { downloadPlugins, localPluginsVersion } from './bundleDefaultPlugins';
+import fs = require('fs-extra');
+const fetch = require('node-fetch');
+
+describe('bundlePlugins', function() {
+
+	beforeEach(() => {
+		jest.setTimeout(20000);
```

Comment:
What is this setTimeout for?

Reply:
I am using this just to increase the timeout for tests on my local system, otherwise they seem to fail randomly. This will be removed before merging the code.